test/system/irohad_test.cpp

```diff
@@ -122,9 +128,26 @@ DROP TABLE IF EXISTS index_by_id_height_asset;
 
  public:
   boost::filesystem::path irohad_executable;
-  std::chrono::milliseconds timeout = 1s;
+  const std::chrono::milliseconds kTimeout = std::chrono::seconds(1);
+  const std::string kAddress;
+  const uint16_t kPort;
 
- private:
+  boost::optional<child> iroha_process_;
+
+  /**
+   * Command client resubscription settings
+   *
+   * The do-while loop imitates client resubscription to the stream. Stream
+   * "expiration" is a valid designed case (see pr #1615 for the details).
+   *
+   * The number of attempts (5) is a magic constant here. The idea behind this
+   * number is the following: five resubscription with 3 seconds timeout is
+   * usually enough to pass the test; if not - most likely there is another bug.
+   */
+  const uint32_t resubscribe_attempts = 5;
+  const std::chrono::seconds resubscribe_timeout = std::chrono::seconds(3);
+
+ protected:
```

Comment:
Why not private?

Reply:
You are right. They can be private. I will change this.